velox/common/memory/tests/FragmentationBenchmark.cpp

```diff
@@ -165,9 +165,9 @@ class FragmentationTest {
   std::string sizeString(size_t size) {
     char str[20];
     if (size < 1 << 20) {
-      sprintf(str, "%ldK", size >> 10);
+      snprintf(str, 20, "%ldK", size >> 10);
```

Comment:
18 since we also need to count for 'K' and `'\0'`?

Reply:
As per snprintf docs it includes the null char , so 20 should be ok imv.

Reply:
NVM I think it should be ok in this case

Reply:
Yeah, accurately speaking, it is 18, but giving porting from sprintf, the alternative way is to just put 20, aka. `sizeof(str)`.

Reply:
Thanks for your scrutiny on the buffer size selection!